pyproject.toml

```diff
@@ -141,6 +142,8 @@ dev = [
     "ruff>=0.9.6",
     "stdlibs>=2024.10.21.16",
     "toml>=0.10.2",
+    "tox>=4.24.1",
+    "tox-uv",
```

Comment:
Why didn't you also fix version of tox-uv?

Reply:
I'm not seeing an error about unresolvable interdependencies anymore, so I've rebased on `main` and fixed `tox-uv` to the latest version.